### PR TITLE
replace all references to WP support with CP forums

### PR DESCRIPTION
**Pull request by senlin**
addresses issue #293 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/readme.html [hunk 1]
```diff
@@ -26,7 +26,7 @@ <h2>Installation: Famous 5-minute install</h2>
 			<li>Open <span class="file"><a href="wp-admin/install.php">wp-admin/install.php</a></span> in your browser.</li>
 		</ol>
 	</li>
-	<li>Once the configuration file is set up, the installer will set up the tables needed for your blog. If there is an error, double check your <code>wp-config.php</code> file, and try again. If it fails again, please go to the <a href="https://wordpress.org/support/" title="WordPress support">support forums</a> with as much data as you can gather.</li>
+	<li>Once the configuration file is set up, the installer will set up the tables needed for your blog. If there is an error, double check your <code>wp-config.php</code> file, and try again. If it fails again, please go to the <a href="https://forums.classicpress.net/" title="WordPress support">support forums</a> with as much data as you can gather.</li>
```
[the remark this answers is not in the document]
Comment (senlin) [reply src/readme.html]: Added them only 8 strings

File: tests/phpunit/tests/theme.php [hunk 2]
```diff
@@ -9,7 +9,7 @@ class Tests_Theme extends WP_UnitTestCase {
 	protected $theme_slug = 'twentysixteen';
 	protected $theme_name = 'Twenty Sixteen';
 	protected $default_themes = array(
-		'twentyfifteen', 'twentysixteen', 'twentyseventeen',
+		'twentyfifteen', 'twentysixteen', 'twentyseventeen', 'cp2017-child', 'cp2016-child'
```
Comment (nylen) [inline tests/phpunit/tests/theme.php]: Let's not include this change in this PR, it's part of #292 instead.

**Comment by nylen** [comment]
"This looks good for merge" means "I looked at this and don't see any issues."  It's helpful to know when someone looks at a PR, thanks @bahiirwa.

However I see a few things that need to be fixed first.  I think we should also be linking directly to https://forums.classicpress.net/c/support here, instead of the main forum URL.

Also: how and why were these two files modified?

File: src/wp-admin/includes/plugin-install.php [hunk 3]
```diff
@@ -634,7 +634,7 @@ function install_plugin_information() {
 				) );
 				?>
 				<div class="counter-container">
-						<span class="counter-label"><a href="https://wordpress.org/support/plugin/<?php echo $api->slug; ?>/reviews/?filter=<?php echo $key; ?>"
+						<span class="counter-label"><a href="https://forums.classicpress.net/plugin/<?php echo $api->slug; ?>/reviews/?filter=<?php echo $key; ?>"
```
Comment (nylen) [inline src/wp-admin/includes/plugin-install.php]: This change is incorrect as we do not have subforums for each plugin.

File: tools/i18n/makepot.php [hunk 4]
```diff
@@ -68,14 +68,14 @@ class MakePOT {
 		),
 		'cp-plugin' => array(
 			'description' => 'Translation of the ClassicPress plugin {name} {version} by {author}',
-			'msgid-bugs-address' => 'https://wordpress.org/support/plugin/{slug}',
+			'msgid-bugs-address' => 'https://forums.classicpress.net/plugin/{slug}',
```
Comment (nylen) [inline tools/i18n/makepot.php]: This change is incorrect as we do not have subforums for each plugin.

File: tools/i18n/makepot.php [hunk 5]
```diff
@@ -68,14 +68,14 @@ class MakePOT {
...
 			'copyright-holder' => '{author}',
 			'package-name' => '{name}',
 			'package-version' => '{version}',
 		),
 		'cp-theme' => array(
 			'description' => 'Translation of the ClassicPress theme {name} {version} by {author}',
-			'msgid-bugs-address' => 'https://wordpress.org/support/theme/{slug}',
+			'msgid-bugs-address' => 'https://forums.classicpress.net/theme/{slug}',
```
Comment (nylen) [inline tools/i18n/makepot.php]: This change is incorrect as we do not have subforums for each theme.

File: tests/phpunit/tests/theme.php (hunk 2 above)
Comment (senlin) [reply tests/phpunit/tests/theme.php]: Sorry for that, I don't even know how this got in here?

Now how to get it out?

File: tools/i18n/makepot.php (hunk 4 above)
Comment (invisnet) [reply tools/i18n/makepot.php]: Let's leave it pointing to WP for now - we're using their plugin directory, makes sense to leave support there.

File: tools/i18n/makepot.php (hunk 5 above)
Comment (invisnet) [reply tools/i18n/makepot.php]: Let's leave it pointing to WP for now - we're using their theme directory, makes sense to leave support there.

File: src/wp-admin/includes/plugin-install.php (hunk 3 above)
Comment (invisnet) [reply src/wp-admin/includes/plugin-install.php]: Let's leave this one pointing to WP too.

File: tools/i18n/makepot.php (hunk 5 above)
Comment (invisnet) [reply tools/i18n/makepot.php]: The description should probably still say WordPress too.

File: tools/i18n/makepot.php (hunk 4 above)
Comment (invisnet) [reply tools/i18n/makepot.php]: The description should probably still say WordPress too.

**Comment by senlin** [comment]
> I think we should also be linking directly to https://forums.classicpress.net/c/support here, instead of the main forum URL.

@nylen please discuss this with @Mte90 and make a decision, so we can change it in this PR instead of having to come back to it.

> Also: how and why were these two files modified?

I did a blanket S&R, should I revert those two too?

File: src/wp-admin/includes/dashboard.php [hunk 6]
```diff
@@ -1544,7 +1544,7 @@ function wp_welcome_panel() {
 			/* translators: 1: link to ClassicPress FAQs page, 2: link to ClassicPress support forum */
 			__( 'If you need help with something else, please see our <a href="%1$s"><strong>FAQs page</strong></a>. If your question is not answered there, you can make a new post on our <a href="%2$s"><strong>support forum</strong></a>.' ),
 			'https://docs.classicpress.net/faq-support/',
-			'https://forums.classicpress.net/c/support/'
+			'https://forums.classicpress.net/c/supportc/support/'
```
Comment (nylen) [inline src/wp-admin/includes/dashboard.php]: One last thing to fix in this PR: there are a couple of URLs that now say `/c/supportc/support/`.

File: src/wp-admin/maint/repair.php [hunk 7]
```diff
@@ -144,7 +144,7 @@
 	}
 
 	if ( $problems ) {
-		printf( '<p>' . __('Some database problems could not be repaired. Please copy-and-paste the following list of errors into the <a href="%s">support forum</a> for additional assistance.') . '</p>', __( 'https://forums.classicpress.net/c/support/' ) );
+		printf( '<p>' . __('Some database problems could not be repaired. Please copy-and-paste the following list of errors into the <a href="%s">support forum</a> for additional assistance.') . '</p>', __( 'https://forums.classicpress.net/c/supportc/support/' ) );
```
Comment (nylen) [inline src/wp-admin/maint/repair.php]: One last thing to fix in this PR: there are a couple of URLs that now say `/c/supportc/support/`.

File: src/wp-admin/update-core.php [hunk 8]
```diff
@@ -548,7 +548,7 @@ function do_core_upgrade( $reinstall = false ) {
 				show_message( sprintf(
 					/* translators: URL to support forum */
 					__( 'If you see this message after waiting 15 minutes and trying the update again, please make a post on our <a href="%s">support forum</a>.' ),
-					'https://forums.classicpress.net/c/support/'
+					'https://forums.classicpress.net/c/supportc/support/'
```
Comment (nylen) [inline src/wp-admin/update-core.php]: One last thing to fix in this PR: there are a couple of URLs that now say `/c/supportc/support/`.

File: src/wp-includes/update.php [hunk 9]
```diff
@@ -158,7 +158,7 @@ function wp_version_check( $extra_stats = array(), $force_check = false ) {
 			sprintf(
 				/* translators: %s: support forums URL */
 				__( 'An unexpected error occurred. Something may be wrong with ClassicPress.net or this server&#8217;s configuration. If you continue to have problems, please report this issue in our <a href="%s">support forum</a>.' ),
-				'https://forums.classicpress.net/c/support/'
+				'https://forums.classicpress.net/c/supportc/support/'
```
Comment (nylen) [inline src/wp-includes/update.php]: One last thing to fix in this PR: there are a couple of URLs that now say `/c/supportc/support/`.

**Comment by nylen** [comment]
Okay, two things to fix.  I'm still seeing changes in `tests/phpunit/data/pomo/de_DE-2.8.mo` and `tests/phpunit/data/pomo/overload.mo`.

File: src/wp-admin/includes/dashboard.php (hunk 6 above)
Comment (senlin) [reply src/wp-admin/includes/dashboard.php]: :( bizarre, next time I do a manual S&R, automated clearly keeps messing things up.

**Comment by senlin** [comment]
> Okay, two things to fix. I'm still seeing changes in `tests/phpunit/data/pomo/de_DE-2.8.mo` and `tests/phpunit/data/pomo/overload.mo`.

I have no idea how to properly revert the commits of these two files, sorry.

**Comment by nylen** [comment]
No worries, I've reverted the `.mo` files.  Here's what worked for me (command syntax from [this page](http://nicolasgallagher.com/git-checkout-specific-files-from-another-branch/); details may vary depending on how you have your git remotes set up):

```
git fetch upstream
git checkout upstream/develop -- tests/phpunit/data/pomo/de_DE-2.8.mo
git checkout upstream/develop -- tests/phpunit/data/pomo/overload.mo
git commit
```

**Pull request merged**